sizefashion/src/components/Navbar/Logo.tsx

```diff
@@ -0,0 +1,10 @@
+import whiteLogo from '../../assets/logo_white.png'
+import { Link } from 'react-router-dom'
+
+export const Logo = () => {
+  return (
+    <Link to='/'>
+      <img src={whiteLogo} className='w-48' alt='sizebay' />
```

Comment:
@otaaviostein alt image name is not clear enough. It is Sizebay, yes, but what does it stand for? If a user that relies on a11y opens your webapp, will they understand that this is an image that resembles Sizebay's logo?